### PR TITLE
Dark mode UI UX fixes

### LINKED ISSUE #4637: Dark Mode and other UI/UX Fixes
### Cluster dropdown, project overview, and header menu, provider preset (admin panel), addon (on cluster view) dropdowns:

- [x] Change to  `#363636` dropdown background color 
- [x] Change to `#484848` hover item background color
- [x] Disabled text color: `#7c7c7c` (for example when cluster is starting up -> cluster dropdown items)

### Icon Color Changes
- [x] back arrow should be white (cluster overview, MDs, etc Backups, admin panel -> metering, ...) 
- [x] etcd Backups, Snapshots, Restores trashbin icon should be white
- [x] Members edit and delete icon should be white
- [x] lighten loading spinner to `#7e868f`  (or a similar shade) (for example on cluster view additional cluster information)

### Disabled & Validation Changes
- [x] on cluster delete with enforced settings, disabled text color: `#7c7c7c`
- [x] missing validation for "Create cluster from template" (I can enter "aaa" or something and go to next step, there's no template by that name)
- [x] "Create cluster from template" should be disabled if there's no template available

## Universal
- [x] On MDs when opening/closing the Event card the icon and text "x events" jumps left/right 
- [x] Own projects should be first in project dropdown in header
- [x] Headline inside the project dropdown should be styled as shown here
dropdown
- [x] the vertical space between for example Cluster ID label and the ID and SSH Keys label and the key are different, it should be like Cluster ID for all (issue already exists here #4493) 
image

- [x] Add separator lines to dropdown 
image
- [x] Add separator line above sign out 
 image

- [x]  On hover of Member card/Member headline add arrow, make arrow and headline link to Member page
image

### Opinion Requested
- [ ] Cluster Template Wizard should end only with "Create Template" option" not with "create cluster" option. Alternatively we can prove the option: "Create cluster and Cluster Template"  
- [x] Change Delete Cluster Template dialogue "persist." to "stay." or "remain." (more commonly used words)

### PR DESCRIPTION
**Pull request by Waseem826**
**What this PR does / why we need it**:
Dark mode UI UX fixes.

**Which issue(s) this PR fixes**:

Fixes #4637 

**What type of PR is this?**

/kind bug
/kind design

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Dark mode and general UI UX fixes.
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by Talha-Jamil-TJ — CHANGES_REQUESTED** [review]
Feedback Given

File: src/app/cluster-template/component.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ export class ClusterTemplateComponent implements OnInit, OnChanges, OnDestroy {
     const dialogConfig: MatDialogConfig = {
       data: {
         title: 'Delete Cluster Template',
-        message: `Delete <b>${template.name}</b> cluster template permanently? All clusters created using this template will persist.`,
+        message: `Delete <b>${template.name}</b> cluster template permanently? All clusters created using this template will stay.`,
```
Comment (ahmedwaleedmalik) [inline src/app/cluster-template/component.ts:196]: `All clusters created using this template will stay`

Maybe we can re-word this a bit. Using `stay` here doesn't really make sense.

Comment (Waseem826) [reply src/app/cluster-template/component.ts:196]: Yea, I agree. Do you have a suggestion?

File: src/app/shared/components/add-cluster-from-template-dialog/steps/template/component.ts [hunk 2]
```diff
@@ -66,4 +72,11 @@ export class SelectClusterTemplateComponent implements OnInit, OnDestroy {
     this._unsubscribe.next();
     this._unsubscribe.complete();
   }
+
+  templateDisplayName(templateId: string): string {
+    if (templateId) {
+      return this.templates?.find(template => template.id === templateId)?.name || '';
+    }
+    return templateId;
```
[the remark this answers is not in the document]
Comment (Waseem826) [reply src/app/shared/components/add-cluster-from-template-dialog/steps/template/component.ts:80]: Yes, and that's how we're handling this everywhere.

File: src/app/cluster-template/component.ts (hunk 1 above)
Comment (ahmedwaleedmalik) [reply src/app/cluster-template/component.ts:196]:
```
```suggestion
        message: `Delete <b>${template.name}</b> cluster template permanently? The clusters created using this template will not be deleted.`,
```

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve
/hold for @Talha-Jamil-TJ to take another look

**Review by KhizerRehan — APPROVED** [review]

**Review by Talha-Jamil-TJ — APPROVED** [review]

**Pull request merged**